keyboards/adkb96/adkb96.c

```diff
@@ -0,0 +1,19 @@
+#include "adkb96.h"
+
+#ifdef SWAP_HANDS_ENABLE
```

Comment:
Delete the whole SWAP_HANDS_ENABLE block.

Reply:
Why?  This is useful, split keyboard or not.